### PR TITLE
Make fuzzy add-using matches clearer

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/12985

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
New UI looks like this:

image

File: src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs [hunk 1]
```diff
@@ -187,10 +187,10 @@ private static bool IsHostOrTestWorkspace(Project project)
         }
 
         private async Task FindResultsInAllProjectSymbolsAsync(
```
Comment (jmarolf) [inline src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs]: Should we rename this method now that you don't supply a project?

File: src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.SymbolReferenceFinder.cs [hunk 2]
```diff
@@ -75,9 +75,9 @@ private INamespaceSymbol MapToCompilationNamespaceIfPossible(INamespaceSymbol co
             }
 
             internal Task<List<SymbolReference>> FindInAllSymbolsInProjectAsync(
```
Comment (jmarolf) [inline src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.SymbolReferenceFinder.cs]: Should we rename this method now that you don't supply a project?

Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.SymbolReferenceFinder.cs]: That seems reasonable

**Comment by Pilchie** [comment]
The fact that no tests needed to be changed indicates a problem we should probably clear up at this point...

Otherwise :+1:

I don't *love* the 'NewName - Using' for the menu item, but I understand the desire to keep it horizontally short, and I don't have anything better :(

@kuhlenh @DustinCampbell any ideas for text here?

**Pull request merged**